app/helpers/patternfly_components_helper.rb

```diff
@@ -23,21 +23,20 @@ def title_tag(title)
     end
   end
 
-  def body_tag(body)
-    tag.div class: 'pf-c-alert__description' do
-      tag.p body
-    end
-  end
+  def description_tag(description)
+    return unless description
 
-  def pf_inline_alert(title, body, variant: nil)
-    tag.div class: "pf-c-alert pf-m-#{variant} pf-m-inline" do
-      icon_tag(variant) + title_tag(title) + body_tag(body)
+    tag.div class: 'pf-c-alert__description' do
+      tag.p description
     end
   end
 
-  def pf_inline_alert_plain(title, variant: nil)
-    tag.div class: "pf-c-alert pf-m-#{variant} pf-m-inline pf-m-plain" do
-      icon_tag(variant) + title_tag(title)
+  def pf_inline_alert(title, variant: nil, description: nil, plain: false)
```

Comment:
I thought `pf_inline_alert` and `pf_inline_alert_plain` methods are very close, and I thought it could be merged together, differences are managed by keyword args values.

Reply:
They were the same method initially, I had to split them to satisfy codeclimate. Now they are merged again and codeclimate doesn't complain... ¯\_(ツ)_/¯

Reply:
Well, looks like Codeclimate thinks I am a nicer person, so doesn't complain :zipper_mouth_face:

Reply:
IIRC codeclimate is disabled in this branch? Or not.